x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_remote_es.tsx

```diff
@@ -239,6 +239,19 @@ export const OutputFormRemoteEsSection: React.FunctionComponent<Props> = (props)
           </EuiFormRow>
           {inputs.syncIntegrationsInput.value === true && (
             <>
+              <EuiSpacer size="m" />
+              <EuiFormRow>
+                <EuiSwitch
+                  {...inputs.syncUninstalledIntegrationsInput.props}
+                  data-test-subj="syncUninstalledIntegrationsSwitch"
+                  label={
```

Comment:
nit: we could add a `helpText` to say something like  `If enabled, uninstalled integrations will be uninstalled on the remote Elasticsearch cluster`